cmake/AddFdbTest.cmake

```
@@ -151,23 +151,30 @@ if(NOT WIN32)
   set(TEST_PACKAGE_ADD_DIRECTORIES "" CACHE STRING "A ;-separated list of directories. All files within each directory will be added to the test package")
 endif()
 
-function(create_test_package)
-  if(WIN32)
-    return()
-  endif()
+# This sets up a directory with the correctness files common to all correctness packages.
+# This function should be called with the following arguments:
+#
+# - OUT_DIR the directory where files will be staged
+# - CONTEXT the type of correctness package being built (e.g. 'valgrind correctness')
+function(stage_correctness_package)
+  set(oneValueArgs OUT_DIR CONTEXT)
+  cmake_parse_arguments(STAGE "${options}" "${oneValueArgs}" "${multiValueArgs}" "${ARGN}")
```

```suggestion
  cmake_parse_arguments(STAGE "" "${oneValueArgs}" "" "${ARGN}")
```
If you don't have any options or multi-value args, you probably shouldn't pass non-existing variables. Otherwise someone might set those in a higher scope causing weird (and hard to debug) issues.